From: Paul Boddie <paul@boddie.org.uk>
Date: Tue, 15 Aug 2023 23:50:33 +0200
Subject: Use pdfminer instead of pdfminer3.

It seems that pdfminer3 is an old fork of pdfminer for Python 3.
However, pdfminer now seems to be available for Python 3, and it
is packaged for Debian, too.

moin (2.0+git20230330.c55318e3-1) unstable; urgency=medium

  * Initial release.
---
 src/moin/converters/pdf_in.py | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/moin/converters/pdf_in.py b/src/moin/converters/pdf_in.py
index 891b43b..3684366 100644
--- a/src/moin/converters/pdf_in.py
+++ b/src/moin/converters/pdf_in.py
@@ -9,10 +9,10 @@ import io
 from datetime import datetime, timedelta
 import logging as stdlogging
 
-from pdfminer3.pdfpage import PDFPage
-from pdfminer3.pdfinterp import PDFResourceManager, PDFPageInterpreter
-from pdfminer3.converter import TextConverter
-from pdfminer3.layout import LAParams
+from pdfminer.pdfpage import PDFPage
+from pdfminer.pdfinterp import PDFResourceManager, PDFPageInterpreter
+from pdfminer.converter import TextConverter
+from pdfminer.layout import LAParams
 
 from . import default_registry
 from moin.utils.mime import Type, type_text_plain
@@ -21,8 +21,8 @@ from moin import log
 logging = log.getLogger(__name__)
 
 
-# pdfminer3 creates many unwanted info messages
-stdlogging.getLogger("pdfminer3").setLevel(logging.WARNING)
+# pdfminer creates many unwanted info messages
+stdlogging.getLogger("pdfminer").setLevel(logging.WARNING)
 
 
 LAPARAMS = LAParams()
